### ISSUE TITLE
Translatable permissions buttons

### ISSUE THREAD

**Issue by proudh0n** [TangoAgency/material-intro-screen#62]
Could this option be added? It's strange to have the whole application in one language and the buttons always in english.

Maybe the translation string could be passed as a second option to the neededPermissions and possiblePermissions methods.

Many thanks!

**Comment by bezmian** [comment]
Good idea, we'll add that to todo list. ;)

**Comment by proudh0n** [comment]
@bezmian Thanks! :)

**Comment by volvciklum** [comment]
We also need this feature. Please try to include it into the next release. Thanks.

**Comment by xp-vit** [comment]
Also faced this one. Would love it to be added.

**Comment by radzio** [comment]
@proudh0n  @volvciklum  @xp-vit 

**For now** you can simply override `grant_permissions` ( `mis_grant_permissions` in the upcoming 0.0.6 version). But we can't guarantee that we will not change this string resource name in future it is not recommended solution. 

However, in next 0.0.6 we will provide: 
 - possibility to set default permission button text ( `@StringRes` ) on MaterialIntroActivity
 - possibility to override default permission button text on each slide ( `@StringRes` )  using SlideFragmentBuilder
 - library by default will use  `grant_permissions` ( `mis_grant_permissions` since 0.0.6 version) string resource

@proudh0n  @volvciklum  @xp-vit  it would be great if you could add translations for default  `grant_permissions` text in your languages and create some PRs ;). It will be much easier for other library users if it will have default translations out of the box ;)

So guys what do you think about it? 

Cheers!

**Comment by aexvir** [comment]
@radzio Of course, I'll translate the strings as soon as possible. Many thanks for implementing the feature

**Comment by volvciklum** [comment]
Could this issue be merged/closed during next 1-2 weeks? Any chance?
